config/environments/production.rb

```diff
@@ -11,7 +11,7 @@
   config.action_controller.perform_caching = true
 
   # Disable Rails's static asset server (Apache or nginx will already do this)
```

Comment:
Minor: It would be good to add the line `# unless running on Heroku` as per [this commit][1].

[1]: https://github.com/alphagov/smart-answers/commit/4ab90ea5968f52f89b62597a47a1d7dd78a44a12